nsqd/nsqd.go

```diff
@@ -365,11 +396,26 @@ func (n *NSQD) PersistMetadata() error {
 	f.Sync()
 	f.Close()
 
-	err = atomicRename(tmpFileName, fileName)
+	err = os.Rename(tmpFileName, fileName)
 	if err != nil {
 		return err
 	}
 
+	stat, err := os.Lstat(fileNameID)
+	if err != nil || (stat.Mode()&os.ModeSymlink) == 0 {
```

Comment:
If you're curious about the spacing around `&`, gofmt does this to emphasize the operator precedence. I guess that means I don't need the parentheses, I can remove them if preferred.
https://github.com/golang/go/issues/12720